### PR TITLE
Bribe DAO DRAFT

### PR DESCRIPTION
**Pull request by flipchan**
Draft pr for the bribedao pallet project

### CODE REVIEW

**Comment by dzmitry-lahoda** [comment]
is it worth to review partially?

File: frame/bribedao/Readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+## Bribe DAO
+
+Bribe DAO is a pallet created in order for people to buy bribes and place bribes.  Allowing users to sell there votes and third parties to buy votes
+for a certain amount.  
+
+map UID's based on referendum index
```
Comment (dzmitry-lahoda) [inline frame/bribedao/Readme.md:7]: not sure i got what it means about uid

File: frame/bribedao/Readme.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Buying Votes    
+Any user can buy votes by placing a bid for X amount of votes for Y amount of tokens.
```
Comment (dzmitry-lahoda) [inline frame/bribedao/Readme.md:10]: Looks like DEX

**Comment by dzmitry-lahoda** [comment]
is is ready for review?

**Comment by hussein-aitlahcen** [comment]
I believe the base implementation is OK? Can we move this draft from PR to reviewable?

**Comment by flipchan** [comment]
@hussein-aitlahcen  @dzmitry-lahoda fixed branch problems, pr Moved into #337

**Pull request closed without merging**